### PR TITLE
Add murmur3 to hash module

### PR DESCRIPTION
**Pull request by aadi-stripe**
As per the description, we wanted to use murmur3, hence we'd like to add it to the hash module. We are experimenting with it internally at Stripe for some CI tooling, given its known better performance for fair distribution of a small set of inputs.

### CODE REVIEW

File: go/hashmodule/hashmodule.go [hunk 1]
```diff
@@ -24,26 +24,32 @@ import (
...
-//    md5,
-//    sha1,
-//    sha256,
-//  )
+// hash = module(
+//
+//	  md5,
+//		sha1,
+//		sha256,
+//		murmur3,
```
Comment (sushain97) [inline go/hashmodule/hashmodule.go]: Indentation looks off here?

File: go/hashmodule/hashmodule.go [hunk 2]
```diff
@@ -24,26 +24,32 @@ import (
...
 	return &starlarkstruct.Module{
 		Name: "hash",
 		Members: starlark.StringDict{
-			"md5":    starlark.NewBuiltin("hash.md5", fnHash(md5.New)),
-			"sha1":   starlark.NewBuiltin("hash.sha1", fnHash(sha1.New)),
-			"sha256": starlark.NewBuiltin("hash.sha256", fnHash(sha256.New)),
+			"md5":        starlark.NewBuiltin("hash.md5", fnHash(md5.New)),
+			"sha1":       starlark.NewBuiltin("hash.sha1", fnHash(sha1.New)),
+			"sha256":     starlark.NewBuiltin("hash.sha256", fnHash(sha256.New)),
+			"murmur3_64": starlark.NewBuiltin("hash.murmur3_64", fnHash64(murmur3.New64)),
```
Comment (sushain97) [inline go/hashmodule/hashmodule.go]: `murmur3` above but `murmur3_64` here. Former seems better to me.

**Review by sushain97 — CHANGES_REQUESTED** [review]

File: go/hashmodule/hashmodule.go (hunk 1 above)
Comment (aadi-stripe) [reply go/hashmodule/hashmodule.go]: Ah yeah that's my formatter. Can fix

Comment (aadi-stripe) [reply go/hashmodule/hashmodule.go]: fixed

File: go/hashmodule/hashmodule.go (hunk 2 above)
Comment (aadi-stripe) [reply go/hashmodule/hashmodule.go]: fixed

**Review by sushain97 — APPROVED** [review]

**Review by sushain-stripe — APPROVED** [review]

**Pull request merged**